polylogue/cli/app.py

```
@@ -92,6 +152,141 @@ def _legacy_candidates(root: Path) -> List[Path]:
+    console.print(f"Collapse threshold: {defaults.collapse_threshold}")
+    console.print(f"HTML previews: {'on' if defaults.html_previews else 'off'} (theme: {defaults.html_theme})")
+    console.print("Output directories:")
+    for key, value in output_dirs.items():
+        console.print(f"  {key}: {value}")
+    console.print(f"State DB: {data['statePath']}")
+    console.print(f"Runs DB: {data['runsDb']}")
+    console.print("Watchable providers: " + ", ".join(data["watchProviders"]))
+
+
+def run_complete_cli(args: argparse.Namespace, env: Optional[CommandEnv] = None) -> None:
+    env = env or CommandEnv(ui=create_ui(True))
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)